scripts/build-and-provide-package

```diff
@@ -988,8 +988,17 @@ trunk_release() {
 
     ${SUDO_CMD:-} generate-reprepro-codename "$TRUNK_RELEASE"
 
-    ${SUDO_CMD:-} ${REPREPRO_CMD} -b "${REPOSITORY}" ${REPREPRO_OPTS} --ignore=wrongdistribution copymatched "$TRUNK_RELEASE" "$REPOS" '*'
-    [ $? -eq 0 ] || bailout 1 "Error: Failed to copy packages from ${REPOS} to ${TRUNK_RELEASE}."
+
+    if [ -n "${SKIP_REPREPRO_WRAPPER:-}" ] ; then
+      ${SUDO_CMD:-} ${REPREPRO_CMD} -b "${REPOSITORY}" ${REPREPRO_OPTS} \
+        --ignore=wrongdistribution --ignore=uploaders --ignore=surprisingbinary \
+        include "${REPOSITORY}" "${WORKSPACE}/binaries/"*"_${newest_version}"_${architecture}.changes
```

Comment:
> Don't we need the...

yes, we do need it. But I don't like this approach, I'm creating a function. 

> And are you sure about usage of include ``"${REPOSITORY}"`` here, while we're operating on ``$TRUNK_RELEASE``?

yes, I think we have to use ``$TRUNK_RELEASE``